framework/scripts/compile_commands.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+import sys, os
+import json
+
+lines = [line.rstrip() for line in sys.stdin]
```

Comment:
Well, the filename is always the same, since there is only one caller of that script. It is porcelain! Should we put it somewhere else? I can certainly add a comment block though.